chcemvediet/apps/anonymization/cron.py

```diff
@@ -0,0 +1,32 @@
+from django.core.files.base import ContentFile
+
+from poleno.cron import cron_job
+from poleno.attachments.models import Attachment
+from chcemvediet.apps.inforequests.models import Action
+
+from .models import AttachmentNormalization
+from . import content_types
+
+
+@cron_job(run_every_mins=1)
+def attachment_normalization():
+    attachment = Attachment.objects.attached_to(Action).filter(
+        attachmentnormalization__isnull=True
+    ).first()
```

Comment:
Na druhe zamyslenie. Ak by sme potrebovali volat viacej metod ako `.filter(...)` a `.first()`, tak by taketo zalamovanie zacalo byt hodne neprehladne. Pozrel som zvysny kod a vacsinou zvykneme taketo chainy lamat takto:
```
    attachment = (Attachment.objects.attached_to(Action)
            .filter(attachmentnormalization__isnull=True)
            .first()
            )
```

Reply:
pri takomto zalamovani bez pouzitia `\` mi vyhodi vynimku `IndentationError: unexpected indent`

Reply:
Ano, odsadzoval by som to na 8 medzier. PEP8 je divny v tom, ze sa snazi zarovat ku lavej zatvorke, co sposobuje to, ze kazdy statement ma "hanging" riadky s inym odsacenim:

```
    attachment = (Attachment.objects.attached_to(Action)
                  .filter(attachmentnormalization__isnull=True)
                  .first()
                  )
    some_other_attachment = (Attachment.objects.attached_to(Action)
                             .filter(attachmentnormalization__isnull=True)
                             .first()
                             )
```
Co je imho neprehladne. Taktiez ak je lava zatvorka prilis daleko vravo, tak to spravi skaredy "slizik" vela kratnych riadkov potom.

Zalamujme teda normalne na 8 medzier.

Reply:
:ok: